### PR TITLE
add bootstrap-star-rating v4.0.1

### LINKED ISSUE #7353: [Request] Krajee star rating
Krajee star rating
https://github.com/kartik-v/bootstrap-star-rating
http://plugins.krajee.com/star-rating
Could be really great to add this into cdnjs ! Thx in advance !!!

### PR DESCRIPTION
**Pull request by dannyxx001**
@PeterDaveHello please help me check this pull request, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kartik-v/bootstrap-star-rating
 * Official website (optional, not the repository): http://plugins.krajee.com/star-rating
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 495
   - Count of watchers: 50
   - Count of forks: 194
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with autoupdate config

close #7353, cc @kartik-v

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
img missing, see:
https://github.com/kartik-v/bootstrap-star-rating/blob/v4.0.1/css/star-rating.css#L16

File: ajax/libs/bootstrap-star-rating/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "svg"
+  ],
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-star-rating.git",
+    "basePath": "",
+    "files": [
+      "js/*",
+      "css/*"
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-star-rating/package.json]: Should not use this kind of wildcard in non-dist folder. Please try `"css/blah*"`

Comment (dannyxx001) [inline ajax/libs/bootstrap-star-rating/package.json]: OK, I have updated. Please check again, thanks.

**Comment by PeterDaveHello** [comment]
`js/star-rating_locale_LANG.js` is just a language translation template, should not be added.

**Comment by dannyxx001** [comment]
@PeterDaveHello please help me check this update again, thank you.

File: ajax/libs/bootstrap-star-rating/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "glyphicon",
+    "svg"
+  ],
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-star-rating.git",
+    "basePath": "",
+    "files": [
+      "(js|css)/star-rating!(_locale_LANG.js)",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-star-rating/package.json]: Please separate different pattern rule individually, in this case, if you want to use js/css in the same rule, you should try `star-rating*`, but not mix with lang files.

Comment (dannyxx001) [inline ajax/libs/bootstrap-star-rating/package.json]: I don't know what you mean. Would you please check the two cases I wrote below?
You mean I should write like this?
(1)
js/star-rating!(_locale_LANG)
css/star-rating*
or
(2)
(js|css)/star-rating*
js/star-rating!(_locale_LANG)
or
(3)
both wrong

Thank you.

Comment (PeterDaveHello) [inline ajax/libs/bootstrap-star-rating/package.json]: You don't need to add `_locale` in the exclude part, try to add its own rule like:
`start-rating_locale` ...

Comment (dannyxx001) [inline ajax/libs/bootstrap-star-rating/package.json]: OK, I have updated, but I'm not sure whether I do correctly or not. Please help me check again, thank you. :D

File: ajax/libs/bootstrap-star-rating/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "svg"
+  ],
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-star-rating.git",
+    "basePath": "",
+    "files": [
+      "(js|css)/star-rating*",
+      "js/star-rating_locale_!(LANG)",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-star-rating/package.json]: Try
```js
  "+(js|css)/star-rating.*",
  "js/star-rating_locale_!(LANG)*",
```

Comment (dannyxx001) [inline ajax/libs/bootstrap-star-rating/package.json]: @PeterDaveHello OK, thanks for teaching me, and I have updated it.

**Comment by maruilian11** [comment]
please check duplicated PR next time
thanks

**Comment by dannyxx001** [comment]
Thank you for your reminding. I will pay more attention to it.

**Pull request merged**